### PR TITLE
stylizing the markup of README file

### PR DESCRIPTION
**Pull request by tmm88**
---

**with this PR, i did basically aim at:**

- *improving the markup styling of the README file;*

---

**my overall intention with this was basically to:**

- **improving the overal UI/UX of of the file;**

---

**by doing so**:

- *make the file easier to be be read;*
- *make the file more cohesive;*

---

**I do understand it if:**

- *the file still needs fixing/tweaking to be merged;*

---

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
@tiagmoraismorgado Just wondering why you removed the info on TaskCluster?

**Comment by tiagmoraismorgado** [comment]
**just updated readme:**
- *added task cluster back again*

---

***please note that it had been previously removed by accident***

**Comment by tiagmoraismorgado** [comment]
had also removed some other text by accident which i just made a re-input

File: README.md [hunk 1]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
 
-Each dataset has a corresponding importer script in `bin/` that can be used to download (if it's freely available) and preprocess the dataset. See `bin/import_librivox.py` for an example of how to import and preprocess a large dataset for training with Deep Speech.
+*Each dataset has a corresponding importer script in `bin/` that can be used to download (if it's freely available) and preprocess the dataset. See `bin/import_librivox.py` for an example of how to import and preprocess a large dataset for training with Deep Speech.*
 
-If you've run the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.
+*If you've ran the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

The past perfect of "to run" is "run" not "ran".

File: README.md [hunk 2]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
 
 ## Checkpointing
 
-During training of a model so-called checkpoints will get stored on disk. This takes place at a configurable time interval. The purpose of checkpoints is to allow interruption (also in the case of some unexpected failure) and later continuation of training without losing hours of training time. Resuming from checkpoints happens automatically by just (re)starting training with the same `--checkpoint_dir` of the former run.
+*During training of a model so called checkpoints will get stored on disk. This takes place at a configurable time interval. The purpose of checkpoints is to allow interruption (also in case of some unexpected failure) and later continuation of training without loosing hours of training time. Resuming from checkpoints happens automatically by just (re)starting training with the same **`--checkpoint_dir`** of the former run.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

'so-called' is being used as a modifier of 'checkpoints' thus it should be hyphenated

File: README.md [hunk 3]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
 
-Be aware however that checkpoints are only valid for the same model geometry they had been generated from. In other words: If there are error messages of certain `Tensors` having incompatible dimensions, this is most likely due to an incompatible model change. One usual way out would be to wipe all checkpoint files in the checkpoint directory or changing it before starting the training.
+*Be aware however that checkpoints are only valid for the same model geometry they had been generated from. In other words: If there are error messages of certain `Tensors` having incompatible dimensions, this is most likely due to an incompatible model change. One usual way out would be to wipe all checkpoint files in the checkpoint directory or changing it before starting the training.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 4]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
 
-## Exporting a model for inference
+## Exporting a model for serving
```
Comment (kdavis-mozilla) [inline README.md]: We don't support serving. So this was correct as written

File: README.md [hunk 5]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
 
-If the `--export_dir` parameter is provided, a model will have been exported to this directory during training.
-Refer to the corresponding [README.md](native_client/README.md) for information on building and running a client that can use the exported model.
+*If the **`--export_dir`** parameter is provided, a model will have been exported to this directory during training.
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 6]
```diff
@@ -57,47 +57,44 @@ pip install -r requirements.txt
...
+Refer to the corresponding [README.md](native_client/README.md) for information on building and running a client that can use the exported model.*
 
 ## Distributed computing across more than one machine
 
-DeepSpeech has built-in support for [distributed TensorFlow](https://www.tensorflow.org/deploy/distributed). To get an idea on how this works, you can use the script `bin/run-cluster.sh` for running a cluster with workers just on the local machine.
+*DeepSpeech has built-in support for [distributed TensorFlow](https://www.tensorflow.org/deploy/distributed). To get an idea on how this works, you can use the script `bin/run-cluster.sh` for running a cluster with workers just on the local machine.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 7]
```diff
@@ -115,13 +112,12 @@ with 1 parameter server, and 2 workers with 1 GPU each:
 $ run-cluster.sh 1:2:1 --epoch 10
 ```
 
-Be aware that for the help example to be able to run, you need at least two `CUDA` capable GPUs (2 workers times 1 GPU). The script utilizes environment variable `CUDA_VISIBLE_DEVICES` for `DeepSpeech.py` to see only the provided number of GPUs per worker.
-The script is meant to be a template for your own distributed computing instrumentation. Just modify the startup code for the different servers (workers and parameter servers) accordingly. You could use SSH or something similar for running them on your remote hosts.
+*Be aware that for the help example to be able to run, you need at least two `CUDA` capable GPUs (2 workers times 1 GPU). The script utilizes environment variable `CUDA_VISIBLE_DEVICES` for `DeepSpeech.py` to see only the provided number of GPUs per worker. The script is meant to be a template for your own distributed computing instrumentation. Just modify the startup code for the different servers (workers and parameter servers) accordingly. You could use SSH or something similar for running them on your remote hosts.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 8]
```diff
@@ -115,13 +112,12 @@ with 1 parameter server, and 2 workers with 1 GPU each:
...
 
 ## Documentation
 
-Documentation (incomplete) for the project can be found here: http://deepspeech.readthedocs.io/en/latest/
+*Documentation (incomplete) for the project can be found here: http://deepspeech.readthedocs.io/en/latest/*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 9]
```diff
@@ -115,13 +112,12 @@ with 1 parameter server, and 2 workers with 1 GPU each:
...
 
 ## Contact/Getting Help
 
-First, check out our existing issues and the [FAQ on the wiki](https://github.com/mozilla/DeepSpeech/wiki) to see if your question is answered there. If it's not, and the question is about the code or the project's goals, feel free to open an issue in the repo. If the question is better suited for the FAQ, the team hangs out in the #machinelearning channel on [Mozilla IRC](https://wiki.mozilla.org/IRC), and people there can try to answer/help.
+*First, check out our existing issues and the [FAQ on the wiki](https://github.com/mozilla/DeepSpeech/wiki) to see if your question is answered there. If it's not, and the question is about the code or the project's goals, feel free to open an issue in the repo. If the question is better suited for the FAQ, the team hangs out in the #machinelearning channel on [Mozilla IRC](https://wiki.mozilla.org/IRC), and people there can try to answer/help.*
```
Comment (kdavis-mozilla) [inline README.md]: Why the italics?

File: README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # Project DeepSpeech [![Documentation Status](https://readthedocs.org/projects/deepspeech/badge/?version=latest)](http://deepspeech.readthedocs.io/en/latest/?badge=latest)
 
-Project DeepSpeech is an open source Speech-To-Text engine. It uses a model trained by machine learning techniques, based on [Baidu's Deep Speech research paper](https://arxiv.org/abs/1412.5567). Project DeepSpeech uses Google's [TensorFlow](https://www.tensorflow.org/) project to make the implementation easier.
+***Project DeepSpeech** is an **open source** ***Speech-To-Text engine*** that uses a model trained by ***machine learning techniques***, based on <a href="https://arxiv.org/abs/1412.5567"><b>Baidu's Deep Speech research paper</b></a>. **Project DeepSpeech** uses **Google's** <a href="https://www.tensorflow.org/"><b>Tensorflow</b></a> project to facilitate implementation.*
```
Comment (reuben) [inline README.md]: FWIW, I think the formatting is over the top here, and the current styling is good enough as it is.

File: README.md [hunk 11]
```diff
@@ -61,38 +61,35 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
...
 
 Each dataset has a corresponding importer script in `bin/` that can be used to download (if it's freely available) and preprocess the dataset. See `bin/import_librivox.py` for an example of how to import and preprocess a large dataset for training with Deep Speech.
 
-If you've run the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.
+If you've ran the old importers (in `util/importers/`), they could have removed source files that are needed for the new importers to run. In that case, simply remove the extracted folders and let the importer extract and process the dataset from scratch, and things should work.
```
Comment (kdavis-mozilla) [inline README.md]: As your expressing a **conditional** (_If you've ran..._) and a **result** (_they could have removed source..._) you should use the past perfect of _to run_. The past perfect of _to run_ is _have run_ not _have ran_.

File: README.md [hunk 12]
```diff
@@ -61,38 +61,35 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
...
+During training of a model so called checkpoints will get stored on disk. This takes place at a configurable time interval. The purpose of checkpoints is to allow interruption (also in case of some unexpected failure) and later continuation of training without loosing hours of training time. Resuming from checkpoints happens automatically by just (re)starting training with the same **`--checkpoint_dir`** of the former run.
 
 Be aware however that checkpoints are only valid for the same model geometry they had been generated from. In other words: If there are error messages of certain `Tensors` having incompatible dimensions, this is most likely due to an incompatible model change. One usual way out would be to wipe all checkpoint files in the checkpoint directory or changing it before starting the training.
 
-## Exporting a model for inference
+## Exporting a model for serving
```
Comment (kdavis-mozilla) [inline README.md]: We don't support TensorFlow serving. So this, as I said in the last review, is in error.

**Comment by kdavis-mozilla** [comment]
As this seems to be currently abandoned, I'm closing the PR.

**Pull request closed without merging**